whelk-core/src/main/groovy/whelk/JsonLd.groovy

```diff
@@ -622,41 +622,47 @@ class JsonLd {
         return jsonLd
     }
 
-    /**
-     * Convert a post to card.
-     *
-     */
     Map toCard(Map thing, boolean chipsify = true) {
-        Map lensGroups = displayData.get("lensGroups")
-        Map cardLensGroup = lensGroups.get("cards")
         Map result = [:]
 
-        Map card = removeProperties(thing, cardLensGroup)
-        if (chipsify) {
-            card.each {key, value ->
-                result[key] = toChip(value)
+        Map card = removeProperties(thing, 'cards')
+        // If result is too small, see if a chip would be bigger, then use
```

Comment:
Comment is a bit out of sync with the code. Something like "If result is too small, use chip instead" would probably be nicer.